### PR TITLE
rpc button added

### PR DESCRIPTION
**Pull request by D3CRYPT360**
Added RPC button to redirect to website

**WARNING: This has not been tested from my part, please test before merging thanks :)**

### CODE REVIEW

**Comment by OwOHamper** [comment]
I was thinking of linking users directly to .zip since the website is kinda outdated. Anyways you should edit button title from "Download VALORANT-Rank-Yoinker" to "Download VALORANT rank yoinker".

File: src/rpc.py [hunk 1]
```diff
@@ -67,7 +74,8 @@ def set_rpc(self, presence):
                             large_text=mapText,
                             small_image=agent_img,
                             small_text=agent,
-                            start=time.time()
+                            start=time.time(),
+                            buttons=[{"label": "Download VALORANT Rank Yoinker", "url": url}]
```
Comment (zayKenyon) [inline src/rpc.py:78]: `VALORANT Rank Yoinker` -> `VALORANT rank yoinker`

File: src/rpc.py [hunk 2]
```diff
@@ -111,7 +120,8 @@ def set_rpc(self, presence):
                             large_image=mapImage,
                             large_text=mapText,
                             small_image=str(self.data.get("rank")),
-                            small_text=self.data.get("rank_name")
+                            small_text=self.data.get("rank_name"),
+                            buttons=[{"label": "Download VALORANT Rank Yoinker", "url": url}]
```
Comment (zayKenyon) [inline src/rpc.py:124]: `VALORANT Rank Yoinker` -> `VALORANT rank yoinker`

**Review by zayKenyon — CHANGES_REQUESTED** [review]

**Comment by zayKenyon** [comment]
> I was thinking of linking users directly to .zip since the website is kinda outdated. Anyways you should edit button title from "Download VALORANT-Rank-Yoinker" to "Download VALORANT rank yoinker".

Then we should wait until the website is done. As a user, if a button tried to download a program I'd find that super sketchy.

**Comment by zayKenyon** [comment]
Pending #121

File: src/requestsV.py [hunk 3]
```diff
@@ -43,15 +43,13 @@ def check_version(self):
...
+                    update_now = input("Do you want to update now? (Y/n): ")
+                    if update_now.lower() == "n" or update_now.lower() == "no":
                         return
-                    elif udpate_now.lower() == "y" or udpate_now.lower() == "yes" or udpate_now == "":
+                    elif update_now.lower() == "y" or update_now.lower() == "yes" or update_now == "":
                         self.copy_run_update_script(link)
                         os._exit(1)
-                    else:
-                        print('Invalid input please response with "yes" or "no" ("y", "n") or press enter to update')
-                        return
```
Comment (OwOHamper) [inline src/requestsV.py:54]: Only remove this keep the else and print function

File: main.py [hunk 4]
```diff
@@ -78,7 +78,7 @@ def program_exit(status: int):  # so we don't need to import the entire sys modu
     ErrorSRC = Error(log)
     
     Requests = Requests(version, log, ErrorSRC)
-    Requests.check_version()
+    print(Requests.check_version())
```
Comment (OwOHamper) [inline main.py:81]: In the end also remove this

**Pull request merged**